driverService/src/test/java/com/modsen/driverservice/util/E2ETestData.java

```diff
@@ -0,0 +1,9 @@
+package com.modsen.driverservice.util;
+
+public class E2ETestData {
```

Comment:
Актуально для всех аналогичных классов:
```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class E2ETestData {
```